### PR TITLE
Removed mtools dependency, added tests

### PR DESCRIPTION
**Pull request by kevinadi**
mtools dependency created some issue with timestamp rendering in fruitsalad. For example, the log line:

```
2017-08-25T22:23:49.097+0000 I COMMAND [conn0]
```

was rendered as:

```
2017-08-25T22:23:49.970+0000 I COMMAND [conn0]
```

This created an impression that the server time is moving forward/backward after the log files are processed with fruitsalad.

Removing the mtools dependency seems to fix this issue.

I have also added some basic tests to fruitsalad.

### CODE REVIEW

File: fruitsalad_test.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
+import unittest
+import io
+import sys
+from fruitsalad import FruitSaladTool
```
Comment (stennie) [inline fruitsalad_test.py]: @kevinadi Can you add a note to the README with suggested incantation for tests? I used `nosetests -v fruitsalad_test.py`.

File: fruitsalad_test.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        self.saved_stdout = sys.stdout
+        self.stdout = io.BytesIO()
+        sys.stdout = self.stdout
+
+    def tearDown(self):
+        sys.stdout = self.saved_stdout
+
+    def test_milliseconds(self):
+        ''' Timestamps should be printed correctly '''
+        tool = FruitSaladTool(arg_logfile='test/sample.ts.log', arg_seed='test')
```
Comment (stennie) [inline fruitsalad_test.py]: @kevinadi Might be safer to use a numeric constant for the `random.seed()`. The Python docs imply a non-numeric value might result in spurious failures:
> If a is not None or an int or a long, then hash(a) is used instead. Note that the hash values for some types are nondeterministic when PYTHONHASHSEED is enabled.

File: fruitsalad_test.py [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    def setUp(self):
+        self.saved_stdout = sys.stdout
+        self.stdout = io.BytesIO()
+        sys.stdout = self.stdout
+
+    def tearDown(self):
+        sys.stdout = self.saved_stdout
+
+    def test_milliseconds(self):
+        ''' Timestamps should be printed correctly '''
```
Comment (stennie) [inline fruitsalad_test.py]: Isn't this actually testing that timestamps are not modified?

File: fruitsalad_test.py (hunk 1 above)
Comment (kevinadi) [reply fruitsalad_test.py]: Done

**Pull request merged**